cohorts/utils.py

```diff
@@ -185,3 +185,10 @@ def get_logger(name, level=logging.INFO):
         logger.handlers = []
     logger.setLevel(level)
     return logger
+
+def set_attributes(obj, additional_data):
+    if additional_data is not None:
+        for key, value in additional_data.items():
+            if hasattr(obj, key):
+                raise ValueError("Key %s in additional_data already exists in this object" % key)
+            setattr(obj, _strip_column_name(key), value)
```

Comment:
@jburos I just updated `set_attributes` to remove the `None` check at the beginning, instead moving that to `Sample` and `Patient`. Also added a docstring.